anaconda: Do not disable other repos by default

This reverses commit fee557978f76ed2c88800ed4ea2f33af0a752c9c from
the Anaconda git repository.

This is needed so that enabled repositories that come from product.img
or updates.img are active by default.

Index: anaconda-16.25/pyanaconda/yuminstall.py
===================================================================
--- anaconda-16.25.orig/pyanaconda/yuminstall.py
+++ anaconda-16.25/pyanaconda/yuminstall.py
@@ -586,11 +586,6 @@ class AnacondaYum(yum.YumBase):
             del(repo)
             raise RepoError, "Excluding disabled repo %s for prerelease" % name
 
-        # If repo=/method= was passed in, we want to default these extra
-        # repos to off.
-        if self._baseRepoURL:
-            repo.enabled = False
-
         return repo
 
     def setProxy(self, src, dest):
